src/+otp/+kortewegdevries/jacobianAdjointVectorProduct.m

```diff
@@ -0,0 +1,7 @@
+function du = jacobianAdjointVectorProduct(t, u, v, Theta, Alpha, Nu, Rho, Dx, Dxt, D3xt)
```

Comment:
the file name is wrong. should be all lower case

Reply:
Lorenz63, Lorenz96, QG all use camelCase